konig-schemagen/src/main/java/io/konig/schemagen/sql/SqlTableGenerator.java

```diff
@@ -68,6 +69,23 @@ public SqlTable generateTable(Shape shape) throws SchemaGeneratorException {
 	}
 
 
+	private void addIdColumn(Shape shape, SqlTable table) {
+		SqlKeyType keyType = SqlKeyType.PRIMARY_KEY;
+		if(shape.getNodeKind() == NodeKind.IRI){
+			for (PropertyConstraint p : shape.getProperty()) {
+				if (p.getStereotype() != null) {
+					if(p.getStereotype().getLocalName().contains("primaryKey")){
```

Comment:
There's no real benefit to having the second `if` statement nested inside the first.  As a general rule, it would be better to combine them into a single `if` statement using the `&&` operator.